control/control.SLES.xml

```diff
@@ -729,6 +729,12 @@ Please visit us at http://www.suse.com/.
                     <enable_next>yes</enable_next>
                 </module>
                 <module>
+                    <label>Installer Update</label>
+                    <name>update_installer</name>
+                    <enable_back>yes</enable_back>
+                    <enable_next>yes</enable_next>
```

Comment:
The default is to have Back and Next enabled, I have verified it with the System Role step recently. But it is not documented :-/